app/lib/ui/views/active/atr_editing_screen.dart

```diff
@@ -185,6 +185,46 @@ class _ATREditingScreenState extends State<ATREditingScreen> {
                         style: TextStyle(
                             color: NavyBlue, fontWeight: FontWeight.bold),
                       ),
+                      actions: [
+                        IconButton(
+                            icon: Icon(Icons.delete),
+                            color: NavyBlue,
+                            onPressed: () => showDialog(
+                                context: context,
+                                child: AlertDialog(
+                                  title: Text(
+                                    'Confirm Delete',
+                                    style: DialogBoxTitleStyle,
+                                  ),
+                                  content: Text(
+                                      'Are you sure you want to delete this record?',
+                                      style: DialogBoxContentStyle),
+                                  actions: [
+                                    FlatButton(
+                                      child: Text(
+                                        'Cancel',
+                                        style: TextStyle(
+                                            fontWeight: FontWeight.bold),
+                                      ),
+                                      onPressed: () => Navigator.of(context,
```

Comment:
- [ ] You should use the `model`'s functions for navigation using `NavigationService`. We don't use this way anymore.